### PR TITLE
docs: fix typed example not to fail against Akka 2.6 and/or Scala 2.11

### PR DESCRIPTION
**Pull request by jrudolph**
Refs #2676

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -108,6 +108,21 @@ def add213CrossDirs(config: Configuration): Seq[Setting[_]] = Seq(
   }
 )
 
+// TODO: remove once 2.11 is dropped
```
Comment (raboof) [inline build.sbt]:
```suggestion
// TODO: remove once 2.11 is dropped, https://github.com/akka/akka-http/issues/2589
```

File: build.sbt [hunk 2]
```diff
@@ -108,6 +108,21 @@ def add213CrossDirs(config: Configuration): Seq[Setting[_]] = Seq(
...
+/**
+ * Adds a `src/.../scala-2.13+` source directory for Scala 2.13 and newer
```
Comment (raboof) [inline build.sbt]:
```suggestion
 * Adds a `src/.../scala-2.12+` source directory for Scala 2.12 and newer
```

File: build.sbt [hunk 3]
```diff
@@ -108,6 +108,21 @@ def add213CrossDirs(config: Configuration): Seq[Setting[_]] = Seq(
...
+ * and a `src/.../scala-2.13-` source directory for Scala version older than 2.13
```
Comment (raboof) [inline build.sbt]:
```suggestion
 * and a `src/.../scala-2.12-` source directory for Scala version older than 2.12
```

**Comment by jrudolph** [comment]
Thanks for the fixes, @raboof.

**Pull request merged**